src/Text/LLVM/PP.hs

```diff
@@ -227,6 +230,18 @@ ppLayoutBody size abi mb = int size <> char ':' <> int abi <> pref
     Nothing -> empty
     Just p  -> char ':' <> int p
 
+-- | Pretty-print the common case for data layout specifications.
+ppLayoutBody' :: Int -> Int -> Maybe Int -> Fmt (Maybe Int)
+ppLayoutBody' size abi mb mg = int size <> char ':' <> int abi <> pref <> gep
+  where
+  pref = case mb of
+    Nothing -> empty
+    Just p  -> char ':' <> int p
+  gep = case (mb, mg) of
+    (Just _, Just g) -> char ':' <> int g
+    _ -> empty
```

Comment:
We now have two functions for pretty-printing data layout bodies (`ppLayoutBody` and `ppLayoutBody'`), both of which claim to be "the common case" in their Haddocks. This seems somewhat suspect. Instead of having two functions, could we simply add a `Maybe Int` argument to `ppLayoutBody`?